libs/usb-mocking/README.md

```diff
@@ -1,52 +1,52 @@
 # USB Drive Mocking
 
-The `bash` scripts in this library are useful for mocking USB drives in
-`vxsuite`. The mock consists of a simple loop device backed by an `.img` file.
-The contained filesystem persists and its contents can be separately inspected
-and manipulated.
+The scripts in this library are useful for mocking USB drives in `vxsuite`. The
+mock consists of a simple loop device backed by an `.img` file. The contained
+filesystem persists and its contents can be separately inspected and
+manipulated.
 
 The current iteration of these scripts only supports one virtual USB drive image
 at a time. You can reset and resize whenever necessary.
 
-### `insert.sh` - Insert Drive
+### Insert Drive
 
 "Inserts" the virtual USB drive. Does not mount the drive.
 
 ```bash
-sudo ./insert.sh
+pnpm usb-insert
```

Comment:
This will conflict with #3314, which relies directly on the previous scripts.

Wondering if it would make sense to provide a TS API for the dev dock to use...

Reply:
It would be nice to have a TS API.

If that's not possible or easy for whatever reason, I think we can just substitute the `bash` commands for `pnpm` commands.